common/src/main/java/com/microsoft/identity/common/internal/request/MsalBrokerRequestAdapter.java

```diff
@@ -257,8 +266,48 @@ public BrokerAcquireTokenSilentOperationParameters brokerSilentParametersFromBun
         return new HashSet<>(Arrays.asList(scopes));
     }
 
+    public static Bundle getSilentBrokerRequestBundle(final AcquireTokenSilentOperationParameters parameters) {
+        final MsalBrokerRequestAdapter msalBrokerRequestAdapter = new MsalBrokerRequestAdapter();
+
+        final Bundle requestBundle = new Bundle();
+        final BrokerRequest brokerRequest = msalBrokerRequestAdapter.
+                brokerRequestFromSilentOperationParameters(parameters);
+
+        requestBundle.putString(
+                AuthenticationConstants.Broker.BROKER_REQUEST_V2,
+                new Gson().toJson(brokerRequest, BrokerRequest.class)
+        );
+
+        requestBundle.putInt(
```

Comment:
not part of this change, but we should revisit why `CALLER_INFO_UID` is not part of `BrokerRequest`, could be a miss on my part when i was implementing this

Reply:
Do you think it makes sense to set CALLER_INFO_UID to every request bundle? 

(Note: this value is already obtainable through Binder.getCallerUid and AccountManager's bundle.)

Reply:
yeah I think this is not needed as we are explicity adding anyway her e: https://github.com/AzureAD/ad-accounts-for-android/blob/dev/AADAuthenticator/src/msal/java/com/microsoft/identity/client/MicrosoftAuthService.java#L293

Reply:
Keeping it for now because we have a validation logic that check this in Broker.